packages/docs-reanimated/docs/guides/migration-from-3.x.md

```diff
@@ -84,6 +84,53 @@ In Reanimated 4, we renamed `useScrollViewOffset` to `useScrollOffset`. For the
 
 `useAnimatedGestureHandler` was marked as deprecated in Reanimated 3 and has been removed in Reanimated 4. You need to refactor all its usages to the new `Gesture` API introduced in Gesture Handler 2 according to the steps described in [React Native Gesture Handler documentation](https://docs.swmansion.com/react-native-gesture-handler/docs/guides/upgrading-to-2).
 
+### Removed `adapters` parameter from `useAnimatedProps`
+
+In Reanimated 4, the `adapters` parameter has been removed from `useAnimatedProps`. All properties that previously required adapters now should work directly.
```

Comment:
TODO: ask @jakex7 what is the first version of RNSVG that doesn't need prop adapters
 ```suggestion
In Reanimated 4, `createAnimatedPropsAdapter` function along with `adapters` parameter in `useAnimatedProps` hook have been removed. If some property needs preprocessing, it must be done inside `useAnimatedProps` worklet.

The most common use case for custom prop adapters was animating `fill` and `stroke` properties of `react-native-svg` components. Note that starting from `react-native-svg@x.y.z` these props no longer need preprocessing and can be animated directly just like other color props like `backgroundColor` or `borderColor`.
```